bundles/org.openhab.binding.dominoswiss/src/main/java/org/openhab/binding/dominoswiss/internal/EGateHandler.java

```diff
@@ -204,6 +214,7 @@ private synchronized void sendCommand(String command, int timeout) {
         Socket localEGateSocket = egateSocket;
```

Comment:
Please check the method sendCommand, the parameter "timeout" is apparently not used. To be removed if not required.

Reply:
@lolodomo do  you know how I can switch from openhab-addons "main" branch to OH 3.4 branch? My bugfix is for OH 3.4 - now when I build the Binding I get a 4.0.0 Snapshot which I can not test in my 3.4 Openhab Installation. Thanks a lot for any hint.

Reply:
If you want to build a 3.4.x version with Java 11, you will have to create a branch from the 3.4.x branch and add your new code.
But note that any PR should now be applied to the main branch (OH 4 using Java 17).

Reply:
> fixed, it is required and is now used.

But new code not yet published ?

Reply:
Sorry had first to install OH 4 and test it.  Now its published